docs/crio.8.md

```diff
@@ -25,6 +25,7 @@ crio
 [--log=[value]]
 [--log-format value]
 [--log-level value]
+[--log-to-journald]
```

Comment:
Seems a bit long, I would use `log-journald` or simply `journald` to enable logs to be sent to the journal.

Reply:
I agree, I'm pretty iffy about the verbosity too.

Reply:
opted for log-journald because it seems clear but shorter!